firebase/src/main/java/com/rudderstack/android/integration/firebase/Utils.java

```diff
@@ -239,4 +239,42 @@ public static boolean isEmpty(Object value) {
         }
         return false;
     }
+
+    static void attachProperties(android.os.Bundle params, Map<String, Object> properties, boolean filterReservedKeywords) {
```

Comment:
Rename the method name to `attachCustomProperties` or `attachAllCustomProperties`. `Custom` word highlights the nature of these properties!